web/module/pwsExamExtras.jsp

```diff
@@ -1,13 +1,14 @@
             <div id="examExtras">
                         <div class="examExtraCheckbox">
                             <c:choose>
-                                <c:if test="${Special_Need == 'Y'}">
-                                    <input type="checkbox" name="Special_Need" value="Y" checked disabled/>Special Need Child<br/>
-                                </c:if>
+                                <c:when test="${Special_Need == 'Y'}">
+                                    <input type="checkbox" name="Special_Need" value="yes" checked />Special Need Child<br/>
+                                </c:when>
                                 <c:otherwise>
-                                    <input type="checkbox" name="Special_Need" value="Y"/>Special Need Child<br/>
+                                    <input type="checkbox" name="Special_Need" value="yes"/>Special Need Child<br/>
                                 </c:otherwise>
                             </c:choose>

```

Comment:
I know some of this was existing code, but I like to see this block of code reduced to a single line of code like this:
<input type="checkbox" name="Special_Need" value="yes" ${Special_Need == 'Y' ? 'checked' : ''}/>Special Need Child<br/>